### PR TITLE
Vv 141 managers flow refactor gateway code

### PR DESCRIPTION
**Pull request by nax2uk**
# What
-  Refactored code to separate into gateway and use cases
-  added unit tests
# Why
- To enable easier migration to MsSQL
# Screenshots
N/A
# Notes
- separated ticket has been opened for contract tests

### CODE REVIEW

**Review by pl-madetech — APPROVED** [review]
Thanks Azlina all good.

File: src/gateways/MsSQL/updateManagerStatusByUuid.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const updateManagerStatusByUuidGateway = async (db, uuid, status) => {
```
[the remark this answers is not in the document]
Comment (nax2uk) [reply src/gateways/MsSQL/updateManagerStatusByUuid.js]: Got you. This has been done. 🙂

File: src/usecases/archiveManagerByUuid.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import logger from "../../logger";
+
+const archiveManagerByUuid = ({
+  getMsSqlConnPool,
```
[the remark this answers is not in the document]
Comment (nax2uk) [reply src/usecases/archiveManagerByUuid.js]: Got you. This has been done. 🙂

**Comment by nax2uk** [comment]
Thank you Paulo.

**Pull request merged**